.github/workflows/run-tests-command.yml

```diff
@@ -363,6 +363,11 @@ jobs:
       with:
         version: v2.5.0
         token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Free Disk Space (Ubuntu)
```

Comment:
This is necessary because the AWS Js tests were already close to running the runners out of disk space (see https://github.com/pulumi/examples/actions/runs/8060515253/job/22016676190 for example, where we did run out of disk space), and using the dev dependency seems to use just a little bit more disk space, so the failing is more consistent. Since we're already close to running the runner our of disk space this seems like a good thing to do either way.